utils/build-script-impl

```diff
@@ -1542,7 +1542,7 @@ for host in "${ALL_HOSTS[@]}"; do
                 if [ "${BUILD_LLVM}" == "0" ] ; then
                     build_targets=(clean)
                 fi
-                if [[ "${SKIP_BUILD}" || "${SKIP_BUILD_LLVM}" ]] ; then
+                if [[ ( "${SKIP_BUILD}" || "${SKIP_BUILD_LLVM}" ) && "${SWIFT_INCLUDE_TESTS}" == "1" ]] ; then
                     # We can't skip the build completely because the standalone
```

Comment:
I am dubious that this is correct. The reason why is that the standalone that is being referred to here is building swift out of tree of LLVM, not a standalone stdlib.

Given that is true, we also need some parts of LLVM to build swift. Namely, the tblgen, clang-resource-headers and intrinsics_gen. My thought is the right thing to do here is to perhaps check if we are compiling with the HOST CMAKE in some way. We are already passing in that variable by hand to cmake. Maybe we add that as an official build-script option? Then you could check that here and know 100% you are right.